Library/include/CSP/Systems/Users/UserSystem.h

```diff
@@ -162,8 +162,10 @@ class CSP_API CSP_NO_DISPOSE UserSystem : public SystemBase
 	CSP_ASYNC_RESULT void ConfirmUserEmail(NullResultCallback Callback);
 
 	/// @brief Reset the users password.
+	/// @param Token csp::common::String : Bearer Token of the user changing their password
```

Comment:
I think of bearer token as the OAuth access token in the headers. Is that what this token is or is it something else?

Reply:
Good catch. According to swagger this is actually a token that is received via an automated email that is sent

Reply:
I thought that might be what this was. I think we should update the comment to make that more clear before shipping this.

Reply:
sorted